### DISCUSSION TITLE
Will the maintainers also support Open Source Cloud Foundry?

### DISCUSSION THREAD

**Discussion by norman-abramovitz** [CrowdStrike/falcon-boshrelease#4] [Q&A]
Tanzu for VMs is a productization of [Open Source Cloud Foundry](https://www.cloudfoundry.org/).   Are the maintainers also open to supporting Open Source Cloud Foundry as well?  When there are differences, it is handled by [ops files](https://bosh.io/docs/cli-ops-files/).

**Comment by redhatrises** [comment]
Hello,

Support for Open Source Cloud Foundry is currently not on the roadmap, please connect with your TAM over this interest.
